slippymap/slippymap.go

```diff
@@ -116,25 +109,19 @@ func (sm *SlippyMap) Update(deltaOffsetX, deltaOffsetY int, forceUpdate bool) {
 	}
 
 	// tile reposition & alpha increase if needed
-	wgAlpha := sync.WaitGroup{}
-	for _, tile := range sm.tiles {
-		wgAlpha.Add(1)
-		go func(t *MapTile) {
```

Comment:
I removed the goroutine here because it's only doing a couple of maths operations. Using goroutines here is likely a performance negative. Not a bug, just unnecessary I think.

Reply:
Tagging @bluntelk in as he did the goroutine stuff.

Reply:
again, if there is little perf benefit, by all means, let's remove it